src/test/java/com/seller/panel/data/TestDataMaker.java

```diff
@@ -6,7 +6,7 @@
     public static final String X_COMPANY = "X-COMPANY";
     public static final String CONTENT_TYPE = "Content-Type";
     public static final String COMPANY_ID = "DEF";
-    public static final String REGISTER_ENDPOINT = "/api/v1/register";
```

Comment:
I would advise to create separate static final class for endpoints and should be kept in main resources, which can be then used in both controller and during testing.